src/java/org/apache/cassandra/schema/TableParams.java

```diff
@@ -351,7 +351,7 @@ public void appendCqlTo(CqlBuilder builder, boolean isView)
                .append("AND read_repair = ").appendWithSingleQuotes(readRepair.toString())
                .newLine()
                .append("AND speculative_retry = ").appendWithSingleQuotes(speculativeRetry.toString());
-        if (DatabaseDescriptor.getAutoRepairConfig().isAutoRepairSchedulingEnabled())
+        if (DatabaseDescriptor.getRawConfig() != null && DatabaseDescriptor.getAutoRepairConfig().isAutoRepairSchedulingEnabled())
```

Comment:
tiny nit: mind adding a whitespace above this line?
```suggestion

        if (DatabaseDescriptor.getRawConfig() != null && DatabaseDescriptor.getAutoRepairConfig().isAutoRepairSchedulingEnabled())
```